### PR TITLE
B lorient615

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient615.xml [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+ወለተ፡ ሚካኤል፡</persName></foreign> e. g. on <locus target="#111v">f. 111v</locus>, but also erased.</desc>
+</item>
+
+<item xml:id="a4">
+<locus target="#79r"/>
+<desc type="Supplication"/>
+<q xml:lang="gez"><seg part="I">በረከቱ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ወሊቀ፡ ሐራ። ይኩኖ፡ ለንጉሥነ፡ በካፋ፡ ዘመዊእ፡ ጌራ። ወያብኦ፡ ሀገረ፡ ዓይነ፡ ንስር፡ እለ፡ ኢነጸራ።
+ነፍሳተ፡ ጻድቃን፡ ኀበ፡ የሐድሩ። አመ፡ ይሠራዕ፡ ምሳሐ፡ ለጽዮን፡ ማኅበራ። ወዓዲ፡ ዘአጽሐፈ<add place="above">ት</add>፡ ዘንተ፡ መጽሐፈ፡ ዓመተ፡ ድንግል፡ በእንተ፡ ፍቅሩ፡<sic/> ለሚካኤል፡ መልአከ፡ ቅዳሴ፡ ይኅድር።
+በረከቱ፡ ላዕሌሀ፡</seg></q>
+<note>According to the supplication, the manuscript was commissioned by <foreign xml:lang="gez"><persName ref="PRS13462AmataD" role="patron">ዓመተ፡ ድንግል፡</persName></foreign>during the reign
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient615.xml:270]: I would delete the `<note>` and transfer the text inside `<desc>`

File: LondonBritishLibrary/orient/BLorient615.xml [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+<item xml:id="a2">
+<locus target="179v"/>
+<desc type="MixedNote">List of drugs</desc>
+<q xml:lang="am"><seg part="I">ኈልቈ፡ መድኃኒት፤ ዕፀ፡ ምናሄ፤ ዕፀ፡ ዘዌ፤ አሜራ፤ ሰመግ፤ ፅፍሬና፤ ዘርጫ፡ የምድር፡ እምባይ፤ በቶ፤ አሰረት፤ የምድር፡ ቁግ፤ የመካን፡ እንዶድ፡ ስር፤</seg></q>
+</item>
+
+<item xml:id="a3">
+<locus target="#111v"/>
+<desc type="Supplication">Name of the first owner erased; earliest substitution <foreign xml:lang="gez"><persName role="owner" ref="PRS13461Anagda">አናግደ፡
+ወለተ፡ ሚካኤል፡</persName></foreign> e. g. on <locus target="#111v">f. 111v</locus>, but also erased.</desc>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient615.xml:261]: I think it's really better not to write insice `<locus>` in such cases, so as not increasing already existing f./fol. inconsistencies (also above)

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: LondonBritishLibrary/orient/BLorient615.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient615.xml:261]: I hope it's better now.

**Review by thea-m — APPROVED** [review]
I had meant it the other way around - `<locus="#111v"/>` without content - but it's fine anyway, thanks!

**Comment by thea-m** [comment]
Could you please also solve the conflict in the web editor? As you know the manuscript best, that will the fastest way. Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**